packages/contracts-bedrock/src/L1/opcm/OPContractsManagerV2.sol

```diff
@@ -814,6 +885,27 @@ contract OPContractsManagerV2 is ISemver, OPContractsManagerUtilsCaller {
             )
```

Comment:
Actually, same comment as above, this has the effect of changing the SystemConfig. In theory the SystemConfig is really only being used for the variables that should be the same across all interop chains, but it means a constantly changing state variable every upgrade